packages/react/src/components/AccountOverview/AccountOverview.stories.mdx

```diff
@@ -0,0 +1,160 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import AccountOverview from './AccountOverview.tsx';
+import Button from '../Button';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import {withDesign} from '../../../.storybook/utils.ts';
+import Typography from '../Typography';
+
+
+export const meta = {
+  component: AccountOverview,
+  title: StoryConfig.AccountOverview.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => <AccountOverview {...args} />;
+
+# Menu
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+  - [Incomplete](#incomplete)
+  - [Complete](#complete)
+
+## Overview
+
+Menu.
+
+<Canvas>
+  <Story
+    name="Overview"
+    args={{
+      title: <Typography variant="h5">Welcome Mathew</Typography>,
+      subheader: <Typography variant="body2">Manage your personal information, account security and privacy settings.</Typography>, 
+      user:{
+        image: '/assets/images/avatar-john.svg',
+        name: 'Matthew',
+        email: 'matthew@wso2.com'
+      },
+      accountProgress: 60,
+      accountCompletionStepsTitle: "Complete your Profile. It's at 60%",
+      accountCompletionSteps: [
+        {
+          title: 'Add your email address',
+          description: 'You can add your email address to your profile to receive notifications and updates from us.',
+          illustration: <img src="/assets/images/carousel-illustration.svg" alt="carousel illustration" />,
+          action: <Button variant="outlined">Add Email</Button>
+        },
+        {
+          title: 'Add your phone number',
+          description: 'You can add your phone number to your profile to receive notifications and updates from us.',
+          illustration: <img src="/assets/images/carousel-illustration.svg" alt="carousel illustration" />,
+          action: <Button variant="outlined">Add Phone</Button>
+        },
+        {
+          title: 'Add your address',
+          description: 'You can add your address to your profile to receive notifications and updates from us.',
+          illustration: <img src="/assets/images/carousel-illustration.svg" alt="carousel illustration" />,
+          action: <Button variant="outlined">Add Address</Button>
+        }
+      ],
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable story="Overview" />
+
+## Usage
+
+Import and use the `Button` component in your components as follows.
```

Comment:
Shall we update this?

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/58/commits/4973d88a31c62a594c87052bf927a6d40ed14a9f